docs/guides/using-theme-overrides.md

```diff
@@ -421,3 +461,139 @@ example: true
   </div>
 </div>
 ```
+
+### Examples
+
+#### Use global theme for multiple React trees
+
+If you have multiple `ReactDOM.render` calls and you would like to use the same theme with the same overrides for them, then you have 2 options:
+
+- wrap each `ReactDOM.render` application with an `InstUISettingsProvider` with the correct theme:
+
+  ```js
+  ---
+  example: false
+  ---
+  //ui/themeOverride.js
+  import { canvas } from "@instructure/ui-themes"
+
+  const themeWithOverrides = {
+    ...canvas,
+    colors: {
+      brand: 'red'
+    }
+  }
+
+  export { themeWithOverrides }
+  ```
+
+  ```jsx
+  ---
+  example: false
+  ---
+  //ui/app1/index.js
+  ReactDOM.render(
+    <InstUISettingsProvider theme={themeWithOverrides}>
+      <App1>
+    </InstUISettingsProvider>
+  )
+  ```
+
+  ```jsx
+  ---
+  example: false
+  ---
+  //ui/app2/index.js
+  ReactDOM.render(
+    <InstUISettingsProvider theme={themeWithOverrides}>
+      <App2>
+    </InstUISettingsProvider>
+  )
+  ```
+
+  This approach quickly becomes too cumbersome if you have more than a couple of `ReactDOM.render` calls.
+
+- instead you could achive the same thing with global themes:
+
+```jsx
+---
+example: false
+---
+//ui/init.js - make sure ui/init.js runs before any other app code!
+import { canvas } from "@instructure/ui-themes"
+
+canvas.use({
+  overrides: {
+    colors: {
+      brand: 'red'
+    }
+  }
+})
+```
+
+This will register the `canvas` theme with its overrides as the active theme in the [ThemeRegistry](/#ThemeRegistry) and then every InstUI component - that is not wrapped in an `InstUISettingsProvider` - will use this theme no matter which React tree they are rendered in.
+
+#### Use global theme together with InstUISettingsProvider
+
+You can use the global theming and the application level theming together, however there are some caveats you should be aware of:
+
+- if you globally `.use()` a theme and would like to override that theme in a sub-tree of your application, then make sure that sub-tree does not have a theme provided to any `InstUISettingsProvider` above it in the tree:
```

Comment:
hmm this part is a bit hard to read.. I'd rephrase it like this (is my interpretation correct?):
"if you globally `.use()` a theme and would like to override that theme in a sub-tree of your application then just wrap that part in `InstUISettingsProvider`. Here just specify the overrides, not the full theme:"